runtime/doc/lsp.txt

```diff
@@ -1397,7 +1397,10 @@ format({opts})                                          *vim.lsp.buf.format()*
 
 hover()                                                  *vim.lsp.buf.hover()*
     Displays hover information about the symbol under the cursor in a floating
-    window. Calling the function twice will jump into the floating window.
+    window. The window will be dismissed on cursor move. Calling the function
+    twice will jump into the floating window. Once focused in the floating
+    window, `<C-f>` and `<C-b>` can be used to scroll forward and back, and
```

Comment:
`<C-d>`/`<C-u>` also work for scrolling.

Reply:
> Once focused in the floating window, the content buffer can be scrolled as normal and `q` can be used to dismiss the window.

Does that sound better?

Reply:
I would use "usual" instead of "normal" because of the latter already having a special meaning in vim.